addons/gamification/models/gamification_karma_rank.py

```diff
@@ -50,8 +52,8 @@ def write(self, vals):
         if 'karma_min' in vals:
             after_ranks = self.env['gamification.karma.rank'].search([], order="karma_min DESC").ids
             if previous_ranks != after_ranks:
-                users = self.env['res.users'].sudo().search([('karma', '>', 0)])
+                users = self.env['res.users'].sudo().search([('karma', '>=', max(low, 1))])
```

Comment:
Note that I just noticed that here above it's written:

`low = min(vals['karma_min'], self.karma_min)`

But we can call write on multiple records, so... self.karma_min = :boom: ?
Can you give it a quick look, sorry to be a bother but while we are modifying it let's try to make it work in all use cases :smile:

Reply:
Done!